### PR TITLE
feat(Bonus Pagamenti Digitali): [#175883850,#176166431,#175883914,#175883928] Activate digital payments & add satispay screens

### PR DESCRIPTION
**Pull request by fabriziofff**
## Short description
This pr adds the missing screens for the Satispay onboarding workflow.

SatispayKoTimeout            |  SatispayKoNotFound
:-------------------------:|:-----------------------------:
|  

StartSatispaySearchScreen            |  AddSatispayScreen 
:-------------------------:|:--------------------------:
|  

## List of changes proposed in this pull request
- Adds all the missing screens
- Unlock the "digital payments"

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -806,10 +806,22 @@ wallet:
...
       loadingAdd:
         title: "We are adding your Satispay account\n\nPlease wait a few seconds"
+      add:
+        title: "Do you want to add this Satispay account?"
+      koTimeout:
+        title: "We were unable to contact Satispay"
+        body: "Ripova please"
+      koNotFound:
+        title: "Sorry, we do not know that you have activated Satispay yet"
+        body: "Contact Satipay for activation"
```
Comment (thisisjp) [inline locales/en/index.yml:824]: typo (Satipay)

File: locales/it/index.yml [hunk 2]
```diff
@@ -826,10 +826,22 @@ wallet:
...
       loadingAdd:
         title: "Stiamo aggiungendo il tuo account Satispay\n\nAttendi qualche secondo"
+      add:
+        title: "Vuoi aggiungere questo account Satispay?"
+      koTimeout:
+        title: "Non siamo riusciti a contattare Satispay"
+        body: "Ripova per favore"
+      koNotFound:
+        title: "Spiacenti, non ci risulta che tu abbia ancora attivato Satispay"
+        body: "Contatta Satipay per l’attivazione"
```
Comment (fabriziofff) [inline locales/it/index.yml:844]:
```suggestion
        body: "Contatta Satispay per l’attivazione"
```

File: locales/en/index.yml (hunk 1 above)
Comment (fabriziofff) [inline locales/en/index.yml:824]:
```suggestion
        body: "Contact Satispay for activation"
```

Comment (fabriziofff) [reply locales/en/index.yml]: tnx! changed in 2cd90b2 and a351a6c

File: ts/features/wallet/onboarding/satispay/screens/add/AddSatispayScreen.tsx [hunk 3]
```diff
@@ -29,20 +32,36 @@ import LoadAddSatispayComponent from "./LoadAddSatispayComponent";
...
+          leftButton={cancelButtonProps(props.cancel)}
+          rightButton={confirmButtonProps(
+            () => props.satispay && props.confirm(props.satispay),
+            I18n.t("global.buttons.continue")
+          )}
+        />
+      </SafeAreaView>
+    </BaseScreenComponent>
+  );
+};
```
Comment (Undermaken) [inline ts/features/wallet/onboarding/satispay/screens/add/AddSatispayScreen.tsx:65]: I guess we are not handling the error case while app is adding satispay to the wallets
How to reproduce:
- search for satispay 
- show the satispay info screen
- stop local server (or return an error from the relative API)
- add satispay

It displays always the loading state

Maybe `<LoadAddSatispayComponent />` should listen about wallet insertion errors
`hasError: isError(onboardingSatispayAddingResultSelector(state))`

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]

**Pull request merged**